doc : login for public platforms

arthur740212 opened the pull request.

arthur740212 committed:
doc : login for public platforms

andrew-hirata-playeveryware commented on docs/readme_login.md:
```diff
@@ -0,0 +1,304 @@
+
+### Prerequisites
+
+* EOS Platform Interface must be initilaized before performing a login
```
I think we need to specify what 'init' means here.
i.e. needing to have the EOSManager attached to a scene that is being run.

andrew-hirata-playeveryware commented on docs/readme_login.md:
```diff
@@ -0,0 +1,304 @@
+* Auth Login functions are declared in EOS Auth Interface
+* **Account Portal** (`EOS_LCT_AccountPortal`) and **Persistent Auth** (`EOS_LCT_PersistentAuth`) are the primary Auth Types (`LoginCredentialType`) to login  
+  * **Persistent Auth** will login with credentials of the previous successful **Account Portal** login
+* **Dev Auth**(`EOS_LCT_Developer`) is for quick iteration for developers, which could be done by using the Dev-Auth tool provided with the EOS SDK. Find [this](https://github.com/PlayEveryWare/eos_plugin_for_unity/tree/development/docs/Walkthrough.md) for details
```
Instead of 'Find this for details' perhaps change it to 'Read this for more details.'.

andrew-hirata-playeveryware commented on docs/readme_login.md:
````diff
@@ -0,0 +1,304 @@
+```      
+❗*What happens after login should be tailored to fit the game's flows, such as configure UI, open main scene, etc*   
+*For some fail cases we pass an empty callbackinfo `LoginCallbackInfo blankCallbackInfo = new LoginCallbackInfo()` into callbacks to get better debug visibilty*    
+*They are omitted here to keep the file concise, more details could be found in plugin scripts* 
````
I think we could add some references to the exact script that has the details to aid the user in finding it.

Raxnam commented on docs/readme_login.md:
```diff
@@ -0,0 +1,304 @@
+* **Auth Login**       for authorizing an **Epic Account** and related services 
+* **Connect Login**    for connecting the **User** for one **ProductID (Game)**
+
+More information about the distinction could be found [here](https://dev.epicgames.com/en-US/news/accessing-eos-game-services-with-the-connect-interface#a-brief-summary-of-auth-vs-connect-interfaces)
```
it may be good to link the `login_type_by_platform` for when to use each one

arthur740212 committed:
doc : Review fix

Add Linux into list of platforms
Easy init steps
Link to login type table
Add references to the exact login scripts
